app/models/order_item.rb

```diff
@@ -52,4 +64,11 @@ def service_parameters_raw=(val)
   def parameters_contain_sanitized_values?(val)
     val.to_s.include?("\"#{Catalog::OrderItemSanitizedParameters::MASKED_VALUE}\"")
   end
+
+  def mark_item(msg = nil, level: "info", **opts)
```

Comment:
We probably don't need to default `msg = nil` since it's private and we're always passing in `msg`.

Reply:
Oh yeah, good point. That was there initially before I made it more generic. Removed it for that method.